india_banking_connector/india_banking_connector/doctype/bank_request_log/bank_request_log.py

```
@@ -157,3 +159,86 @@ def create_api_log(
+		log_doc.url = url
+		log_doc.method = method
+		log_doc.payload = encrypt_log(cstr(payload), _encrypt)
+		log_doc.config_details = encrypt_log(format_with_indent(account_config), _encrypt)
+		log_doc.reference_doctype = ref_doctype
+		log_doc.reference_docname = ref_docname
+		log_doc.unique_id = unique_id
+		log_doc.connector = (connector or {}).get("doctype")
+		log_doc.connector_name = (connector or {}).get("name")
+		log_doc.encrypted = 1 if _encrypt else 0
+		log_doc.insert(ignore_permissions=True)
+	except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)